dojo/settings/settings.dist.py

```diff
@@ -843,6 +843,7 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
 DEDUPLICATION_ALGORITHM_PER_PARSER = {
     'Anchore Engine Scan': DEDUPE_ALGO_HASH_CODE,
     'anchore_grype': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL,
+    'Burp REST API': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL,
```

Comment:
shouldn't this mean the `unique_id_from_tool` Finding field should be set somewhere to make this work?

Reply:
yes.
I made this modification, it's already in dev since few releases:
```
                    unique_id_from_tool=str(
                        issue.get("serial_number", "")
                    ),  # the serial number is a good candidate for this attribute
                    vuln_id_from_tool=str(
                        issue.get("type_index", "")
                     ), # the type index is a good candidate for this attribute
```